### DISCUSSION TITLE
Using different image for "dagster-run" pod on k8s

### DISCUSSION THREAD

**Discussion by psarka** [dagster-io/dagster#20783] [Q&A]
When using k8s deployment and materializing an asset, two pods are created: "dagster-run-[...]" and after that "dagster-step-[...]". They both use the same docker image as the code location.

In our case (and I expect this is the norm) this image is heavy, as it contains dependencies of all the assets. It seems excessive to use it for dagster-run pod, as it does not do any computation, but rather looks after the dagster-step pod.

Is it possible to use a lighter image for dagster-run, say only containing dagster-* libs?
If yes, how?

---
(I tried runLauncher.config.image section, but it does not even pass the helm validation, and it is not clear from the description which image it is configuring)

**Comment by vietvudanh** [comment]
I would prefer this as a well, seems no one really care? I use spark so the image is like really large, took times to build & pull to k8s nodes. The assets would have been just a repo and git-sync like how Airflow is doing.
